src/main/java/com/spikes2212/dashboard/ChildNamespace.java

```diff
@@ -4,6 +4,10 @@
 
 import java.util.function.Supplier;
 
+/**
+ * This class represents a "subdirectory" of another {@link ChildNamespace} or {@link RootNamespace} which can store
```

Comment:
change `another {@link ChildNamespace} or {@link RootNamespace}` to `another {@link namespace}`